schema.graphql

```diff
@@ -1,111 +1,131 @@
-    balances: [ERC1155Balance!]! @derivedFrom(field: "token")
-    transfers: [ERC1155Transfer!]! @derivedFrom(field: "token")
+  id: ID!
+  type: Int
+  option: Option
+  claim: Claim
+  contract: ERC1155Contract!
+  identifier: BigInt!
+  uri: String
+  totalSupply: ERC1155Balance!
+  balances: [ERC1155Balance!]! @derivedFrom(field: "token")
+  transfers: [ERC1155Transfer!]! @derivedFrom(field: "token")
 }
 
 type ERC1155Balance @entity {
-    id: ID!
-    contract: ERC1155Contract!
-    token: ERC1155Token!
-    account: Account
-    value: BigDecimal!
-    valueExact: BigInt!
-    transferFromEvent: [ERC1155Transfer!]! @derivedFrom(field: "fromBalance")
-    transferToEvent: [ERC1155Transfer!]! @derivedFrom(field: "toBalance")
+  id: ID!
+  contract: ERC1155Contract!
+  token: ERC1155Token!
+  account: Account
+  value: BigDecimal!
+  valueExact: BigInt!
+  transferFromEvent: [ERC1155Transfer!]! @derivedFrom(field: "fromBalance")
+  transferToEvent: [ERC1155Transfer!]! @derivedFrom(field: "toBalance")
 }
+
 type ERC1155Operator @entity {
-    id: ID!
-    contract: ERC1155Contract!
-    owner: Account!
-    operator: Account!
-    approved: Boolean!
+  id: ID!
+  contract: ERC1155Contract!
+  owner: Account!
+  operator: Account!
+  approved: Boolean!
 }
+
 type ERC1155Transfer implements Event @entity(immutable: true) {
-    id: ID!
-    emitter: Account!
-    transaction: Transaction!
-    timestamp: BigInt!
-    contract: ERC1155Contract!
-    token: ERC1155Token!
-    operator: Account!
-    from: Account
-    fromBalance: ERC1155Balance
-    to: Account
-    toBalance: ERC1155Balance
-    value: BigDecimal!
-    valueExact: BigInt!
+  id: ID!
+  emitter: Account!
+  transaction: Transaction!
+  timestamp: BigInt!
+  contract: ERC1155Contract!
+  token: ERC1155Token!
+  operator: Account!
+  from: Account
+  fromBalance: ERC1155Balance
+  to: Account
+  toBalance: ERC1155Balance
+  value: BigDecimal!
+  valueExact: BigInt!
 }
+
 interface Event {
-    id: ID!
-    transaction: Transaction!
-    emitter: Account!
-    timestamp: BigInt!
+  id: ID!
+  transaction: Transaction!
+  emitter: Account!
+  timestamp: BigInt!
 }
 type Transaction @entity(immutable: true) {
-    id: ID!
-    timestamp: BigInt!
-    blockNumber: BigInt!
-    events: [Event!]! @derivedFrom(field: "transaction")
+  id: ID!
+  timestamp: BigInt!
+  blockNumber: BigInt!
+  events: [Event!]! @derivedFrom(field: "transaction")
 }
 
 type DecimalValue @entity {
-    id: ID!
-    value: BigDecimal!
-    exact: BigInt!
-    decimals: Int!
+  id: ID!
+  value: BigDecimal!
+  exact: BigInt!
+  decimals: Int!
+}
+
+type ValoremDayData @entity {
+  # timestamp rounded to current day by dividing by 86400
+  id: ID!
+  # timestamp rounded to current day by dividing by 86400
+  date: Int!
+  # tvl in USD terms
+  totalValueLockedUSD: BigDecimal!
+  # volume in USD terms
+  volumeUSD: BigDecimal!
+  # fees accrued in USD terms
+  feesAccrued: BigDecimal!
```

Comment:
@thal0x we are going to want to be able to sum/accrue fees and volume over time. Does this schema allow for this?